src/scripts/main.js

```diff
@@ -18,7 +18,30 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+let tree = document.querySelector('#tree');
+
+function createTree(element, data) {
+  const list = document.createElement('ul');
+
+  Object.keys(data).forEach((key) => {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+    list.appendChild(li);
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+  });
+
+  element.appendChild(list);
+
+  return list;
+}
+
+createTree(tree, food);
+
+tree = document.querySelector('#tree');
```

Comment:
The reassignment of `tree` seems redundant since it is being assigned the same value. Consider removing this line to avoid unnecessary operations.